app/views/answers/_answer.json.jbuilder

```diff
@@ -1,4 +1,5 @@
 json.extract! answer, :question_id, :comments_count, :id, :content, :votes_count, :accepted, :created_at, :updated_at
+json.voted Vote.voted(answer, current_user)
```

Comment:
yeah. I don't know if it's better to define the voted method on the resource itself? so that we can do something like user.votes.where(voteable: resource) or even better: `has_many :voters, through: :votes, source: :user` and with that we could `resource.voters.include? user` on the model, we can then add the voters to the eager_loaded objects here. :thought_balloon: